pkg/filters/filters.go

```diff
@@ -112,11 +113,24 @@ outer:
 			filterValue = ""
 		}
 		for labelKey, labelValue := range labels {
-			if labelKey == filterKey && (filterValue == "" || labelValue == filterValue) {
+			if inverse && ((labelKey != filterKey) || !matchPattern(filterKey, labelKey)) && (filterValue == "" || labelValue == filterValue) {
```

Comment:
I think this could be simplified;
```
if (filterValue == "" || labelValue == filterValue) {
   match:=  (labelKey == filterKey || matchPattern(filterKey, labelKey))
  switch {
         case inverse && !match:
            continue outer
         case !inverse && match:
           continue outer
    }
 }
```